### DISCUSSION TITLE
Error font-v versioning

### DISCUSSION THREAD

**Discussion by RosaWagner** [fonttools/fontbakery#3594] [Q&A] [answered]
💔 ERROR: Check for font-v versioning.
com.google.fonts/check/fontv
--- Rationale ---
The git sha1 tagging and dev/release features of Source Foundry `font-v` tool
are awesome and we would love to consider upstreaming the approach into fontmake
someday. For now we only emit a WARN if a given font does not yet follow the
experimental versioning style, but at some point we may start enforcing it.
💔 ERROR Failed with ImportError: Failed to initialize: Cmd('git') failed due to: exit code(1)
cmdline: git version
stderr: 'xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun'

**Comment by felipesanches** [comment, marked as answer]
This seems to be a legitimate ERROR message. It indicates that something is wrong with the setup of the user's system.

I see that this message was included in the fontbakery report posted by Emma at https://github.com/google/fonts/issues/4227#issuecomment-1022338828 So, I think @emmamarichal could take a look at this page to see if the solution describe there works for her: https://stackoverflow.com/questions/52522565/git-is-not-working-after-macos-update-xcrun-error-invalid-active-developer-pa

There it is said that the "problem is that Xcode Command-line Tools needs to be updated."

----

I also see at the bottom of that same post (https://github.com/google/fonts/issues/4227#issuecomment-1022338828) that INFO messages are being excluded from the report.

So, the only reason this check was seem was because it resulted in an ERROR. Otherwise it would have been muted. This in then pretty much pointless.

I would like to suggest to run fontbakery with --loglevel INFO instead of --loglevel WARN, so that INFO messages are also included in the report.

**Reply by RosaWagner** [reply]
Yes you are right. We actually fixed it yesterday because another XCode tool issue appeared while using packager. Emma set up her new computer (intel -> M1) with time machine, and so the XCode commandline tools were not using the right path for M1.

**Reply by RosaWagner** [reply]
Then for your suggestion to show the INFO messages, what context are you talking about? All the time or when there is an error?

**Reply by felipesanches** [reply]
All the time ;-)
